README.md

```diff
@@ -6,5 +6,5 @@ This is a Maven Archetype for setting up an empty Cucumber project.
 mvn archetype:generate                                  \
   -DarchetypeGroupId=io.cucumber                        \
   -DarchetypeArtifactId=cucumber-archetype              \
-  -DarchetypeVersion=2.3.1.2                            \
+  -DarchetypeVersion=2.3.1.3                            \
```

Comment:
Would it be sensible to lock the archetype version to cucumber jvms version?

Reply:
Yes, it would. Thanks!